source/endpoints/plugins/RequiredInfo.py

```diff
@@ -0,0 +1,40 @@
+from flask_restplus import Resource
+
+from endpoints.plugins import plugin_manager, namespace
+
+
+@namespace.route("/")
+class Organizations(Resource):
+    """ Show the all the organizations that have plugins """
+
+    @namespace.doc("list_organizations")
+    def get(self):
+        """ Fetch all the organizations that have plugins installed """
+        return plugin_manager.get_organizations()
+
+
+@namespace.route("/<string:organization>/")
+@namespace.response(404, "Organization  not found")
+@namespace.param("organization", "The name of the organization of the plugin")
+class Plugins(Resource):
+    """ Show the required information of a plugin """
```

Comment:
I do not think this docstring is correct, it lists the plugins for that organisation.